### ISSUE TITLE
Permissions for video

### ISSUE THREAD

**Issue by rcho5840** [motioneye-project/motioneye#3025]
Does anyone know how to solve this issue? I am in 0.42, using a raspberry pi 4 with 2 webcams and a flashdrive attached.

![image](https://github.com/motioneye-project/motioneye/assets/134464812/c91b253e-6d75-4947-9fe0-57d253e3f8d5)
![image](https://github.com/motioneye-project/motioneye/assets/134464812/c91b253e-6d75-4947-9fe0-57d253e3f8d5)

I am having a lot of trouble figuring this out. I have written my target_dir in motion.conf, ran the commands listed below, mounted my flashdrive to a permanent directory. I don't know what I can do at this point.

![image](https://github.com/motioneye-project/motioneye/assets/134464812/af4b99e8-08d7-4590-a5dc-d37ce6301e20)
![image](https://github.com/motioneye-project/motioneye/assets/134464812/af4b99e8-08d7-4590-a5dc-d37ce6301e20)

**Comment by MichaIng** [comment]
You need to run those commands with the mount point of the flash drive as argument. Should be obvious, but from the order you write things, it sounds like you probably ran the commands first (with `/home/pi/Monitor` literally?) and mounted the flash drive afterwards.

**Comment by rcho5840** [comment]
No. I put in /media/anomaly, which is the directory that I wish to store my data. Even with those changes, and the rest of the changes above, I am still getting denied permissions

**Comment by MichaIng** [comment]
Ah, I see you use 0.42.1, please switch to 0.43.1 and try again: https://github.com/motioneye-project/motioneye?tab=readme-ov-file#installation
And verify that the motion user or group has at least execute permissions on any parent directory:
```sh
ls -dl / /media /media/anomaly
```

**Comment by MichaIng** [comment]
You could also mount it with the UID and/or GID of the `motion` user to keep permissions right: https://manpages.debian.org/trixie/mount/mount.8.en.html#Mount_options_for_ntfs

And with the `ntfs-3g` package/driver, the mount option/flag `permissions` allows emulated UNIX permissions on NTFS, i.e. `chmod`/`chown` can then be used. But this is done in userland, so raises CPU utilization on I/O.

Since this issue is somewhat old and related to setup/platform rather than motionEye, I'll close it. Feel free to reopen if it is still at topic.
